src/Sulu/Bundle/ContentBundle/Search/EventListener/ReindexListener.php

```diff
@@ -68,9 +76,24 @@ public function onIndexRebuild(IndexRebuildEvent $event)
 
         $output->writeln('<info>Rebuilding content index</info>');
 
+        $typeMap = $this->baseMetadataFactory->getPhpcrTypeMap();
+
+        $condition = '';
+        foreach ($typeMap as $type) {
+            $phpcrType = $type['phpcr_type'];
+
+            if ($phpcrType !== 'sulu:home' && $phpcrType !== 'sulu:path') {
+                if (!empty($condition)) {
+                    $condition .= ' or ';
+                }
```

Comment:
Looks a bit complicated to me... Would build another array, and do something like `implode(' or ', $phpcrTypes)` instead of using the `$condition` variable.